contributors.md

```diff
@@ -78,6 +78,7 @@ The following people are not part of the development team, but have been contrib
 * Olivier Wervers (oli414) - Remove unused objects command, various bugfixes
 * Christian Schubert (Osmodium) - Ensuring custom user content folders, incl. open folder.
 * (Xkeeper0) - Improved banner tooltips; multiplayer status in toolbar.
+* Andrew Rimpici (Andy608) - Xcode fixes for macOS support.
```

Comment:
There's a 'toolchain' section below. I think it fits your contribution better.

Reply:
Thanks, I just updated the file. I wasn't sure where to put it.